webapp/src/main/resources/application.properties

```diff
@@ -1,4 +1,8 @@
 server.port=8000
 
 ares-api-properties.dataRepositoryProperties.urlPrefix=https://wwwinfo.mfcr.cz/cgi-bin/ares/darv_or.cgi?ico=
-ares-api-properties.dataRepositoryProperties.urlSufix=&xml=0&ver=1.0.2
\ No newline at end of file
+ares-api-properties.dataRepositoryProperties.urlSufix=&xml=0&ver=1.0.2
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/ares
```

Comment:
Toto do zdrojaku aplikace nepatri. Muzete do README.md zminit info o tom, co je potreba udelat pro spusteni aplikace. Problem je v tom, ze zde zverejnujete url, ale zejmena heslo k databazi.

Dobrym zvykem je popsat spusteni v README a spousteni aplikace resit s prepinacem `--spring.config.location`, ktera vam umozni definovat konfguraci mimo aplikace. Nicmene toto tam klidne mejte, ale pridejte si to do testu a definujte si nejaky databazovy test.